wazo_calld/plugins/calls/api.yml

```diff
@@ -315,6 +315,9 @@ definitions:
       all_lines:
         type: boolean
         description: Use all of the lines of the user to make the call (ignored when `line_id` is specified).
+      auto_answer:
```

Comment:
Thins is ambiguous: when `POST /calls` is used, there are two legs who are ringing: first the caller, then the callee. This setting only applies to the caller leg. The key name should reflect this behavior.